src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -464,7 +463,7 @@ def load_arguments(self, _):
         # managed cluster
         c.argument('ssh_key_value', type=file_type, completer=FilesCompleter(), validator=validate_ssh_key_for_update)
         c.argument('load_balancer_managed_outbound_ipv6_count', type=int)
-        c.argument('outbound_type', arg_type=get_enum_type(outbound_types), validator=validate_outbound_type_in_update)
+        c.argument('outbound_type', arg_type=get_enum_type(outbound_types))
```

Comment:
Will rp distinguish the type of vnet (managed or custom) and perform the validation accordingly?